libs/vscode/nx-cloud-view/src/nx-cloud-fix-webview.ts

```diff
@@ -411,8 +412,52 @@ export class NxCloudFixWebview {
 
           const success = await updateSuggestedFix(aiFixId, 'APPLIED');
           if (success) {
-            window.showInformationMessage('Nx Cloud fix applied successfully');
-            commands.executeCommand('nxCloud.refresh');
+            window
+              .showInformationMessage(
+                'Nx Cloud fix applied successfully.',
+                'Fetch & Pull Changes',
+              )
+              .then((result) => {
+                if (result === 'Fetch & Pull Changes') {
+                  try {
+                    const cwd = getNxWorkspacePath();
+                    const targetBranch = data.cipe.branch;
+                    
+                    // Always refresh remotes first
+                    execSync('git fetch origin', { cwd });
+                    
+                    // Check if the target branch exists on remote
+                    try {
+                      execSync(`git rev-parse --verify origin/${targetBranch}`, { cwd });
+                    } catch {
+                      // Branch doesn't exist on remote, exit silently
+                      return;
```

Comment:
How come? Wouldn't we want to give the user the feedback that we failed and they need to try themselves? We should print the full `origin/...` that we tried to pull from because a reason for this failure might be that their remote is not called origin